components/brave_ads/content/browser/creatives/search_result_ad/creative_search_result_ad_mojom_web_page_entities_for_testing.h

```diff
@@ -21,6 +21,15 @@ CreativeSearchResultAdMojomWebPageEntities(
 blink::mojom::WebPagePtr CreativeSearchResultAdMojomWebPage(
     std::vector<std::string_view> excluded_property_names);
 
+std::vector<schema_org::mojom::EntityPtr>
+CreativeSearchResultAdMojomWebPageEntitiesWithProperty(
+    std::string_view property_name,
+    std::string_view property_value);
```

Comment:
nit: not sure we need `property_` because the function name ends with `Property`, your thoughts?